Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -456,14 +463,43 @@ private void StyleRadioButtons()
                     ServerURLTextBox.Cursor = null;
                     QCRadioButton.FontWeight = FontWeights.Regular;
                     QCRadioButton.Foreground = Brushes.Black;
-                    QCRadioButton.IsChecked = false;
                     RQMRadioButton.FontWeight = FontWeights.Regular;
                     RQMRadioButton.Foreground = Brushes.Black;
-                    RQMRadioButton.IsChecked = false;
                     RestAPICheckBox.Visibility = Visibility.Hidden;
-                    RallyRadioButton.IsChecked = false;
                     RallyRadioButton.FontWeight = FontWeights.Regular;
                     RallyRadioButton.Foreground = Brushes.Black;
+                    qTestRadioButton.FontWeight = FontWeights.Regular;
+                    qTestRadioButton.Foreground = Brushes.Black;
+                    break;
+                case GingerCoreNET.ALMLib.ALMIntegration.eALMType.Qtest:
+                    qTestRadioButton.FontWeight = FontWeights.ExtraBold;
+                    qTestRadioButton.Foreground = (SolidColorBrush)FindResource("$SelectionColor_Pink");
+                    RQMLoadConfigPackageButton.Visibility = Visibility.Hidden;
+                    DownloadPackageLink.Visibility = Visibility.Collapsed;
+                    Grid.SetColumnSpan(ServerURLTextBox, 2);
+                    ExampleURLHint.Content = "Example: https://qtest-stage.t-mobile.com/ ";
+                    if (!isServerDetailsCorrect)
+                    {
+                        ServerURLTextBox.IsEnabled = true;
+                        ServerURLTextBox.IsReadOnly = false;
+                    }
+                    else
+                    {
+                        ServerURLTextBox.IsEnabled = false;
+                        ServerURLTextBox.IsReadOnly = true;
+                    }
+                    ServerURLTextBox.Cursor = null;
+                    QCRadioButton.FontWeight = FontWeights.Regular;
+                    QCRadioButton.Foreground = Brushes.Black;
+                    RQMRadioButton.FontWeight = FontWeights.Regular;
+                    RQMRadioButton.Foreground = Brushes.Black;
+                    RestAPICheckBox.Visibility = Visibility.Hidden;
+                    RallyRadioButton.FontWeight = FontWeights.Regular;
+                    RallyRadioButton.Foreground = Brushes.Black;
+                    JiraRadioButton.FontWeight = FontWeights.Regular;
+                    JiraRadioButton.Foreground = Brushes.Black;
+                    break;
+                default:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)